### PR TITLE
Internal Merge and Disable package building

### PR DESCRIPTION
**Pull request by Anipik**

### CODE REVIEW

File: eng/restore/harvestPackages.targets [hunk 1]
```diff
@@ -23,6 +23,7 @@
 
     <!-- Allow to override package download and versions in case there is already a PackageDownload set -->
     <ItemGroup>
+      <PackageDownload Include="System.Text.Encodings.Web" Version="4.5.1" />
```
Comment (ViktorHofer) [inline eng/restore/harvestPackages.targets:26]: Why was this necessary?

**Review by ViktorHofer — APPROVED** [review]

File: eng/restore/harvestPackages.targets (hunk 1 above)
Comment (Anipik) [reply eng/restore/harvestPackages.targets:26]: This was necessary to avoid harvesting from 4.7.1 package. I think we can delete this now cc @safern
